pkg/client/client.go

```diff
@@ -611,16 +634,46 @@ func string2dysk(asstring string) (*Dysk, error) {
 	return &d, nil
 }
 
-// Dysk as string
-func dysk2string(d *Dysk) string {
+func (c *dyskclient) getDyskSas(d *Dysk) (string, error) {
+	pageBlob, err := c.pageblob_get(d)
+	if nil != err {
+		return "", err
+	}
+
+	//FifteenMin := time.Minute * time.Duration(-15) // allow clock skew https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1
+
+	options := storage.BlobSASOptions{}
+	options.APIVersion = "2017-04-17"
+	//options.Start = time.Now().Add(FifteenMin)
```

Comment:
You are doing the right thing and can remove the code. The best solution for the clock skew is not setting the start time.